packages/cli-plugin-mongoose/templates/config.hbs

```diff
@@ -1,8 +1,5 @@
 export default {
   id: "{{name}}",
   url: process.env.{{symbolName}}_URL || "mongodb://localhost:27017/{{name}}",
```

Comment:
Also, this is populated as `DEFAULT_URL` which doesn't seem like the best name for env var for mongo. 
`MONGO_URL` or `DEFAULT_MONGO_URL` might be better?

Reply:
it depend. If you have multiple mongoose connection, the env variable must be different per connection.

Reply:
Yup, `{{symbolName}}_MONGO_URL` would help with that right?
Or use a different symbol name (`MONGO`) when generating the initial project.
If a user is explicitly generating new configs, that's fine as they are already somewhat familiar with the framework.

Reply:
DEFAULT_URL in env can mean a different number of things.
DEFAULT_MONGO_URL is more specific.